src/dispatch/case_cost_type/views.py

```diff
@@ -0,0 +1,105 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+
+from dispatch.database.core import DbSession
+from dispatch.database.service import CommonParameters, search_filter_sort_paginate
+from dispatch.auth.permissions import SensitiveProjectActionPermission, PermissionsDependency
+from dispatch.models import PrimaryKey
+
+from .models import (
+    CaseCostTypeCreate,
+    CaseCostTypePagination,
+    CaseCostTypeRead,
+    CaseCostTypeUpdate,
+)
+from .service import create, delete, get, update
+
+
+router = APIRouter()
+
+
+@router.get("", response_model=CaseCostTypePagination)
+def get_case_cost_types(common: CommonParameters):
+    """Get all case cost types, or only those matching a given search term."""
+    return search_filter_sort_paginate(model="CaseCostType", **common)
+
+
+@router.get("/{case_cost_type_id}", response_model=CaseCostTypeRead)
+def get_case_cost_type(db_session: DbSession, case_cost_type_id: PrimaryKey):
+    """Get an case cost type by its id."""
```

Comment:
```suggestion
    """Get a case cost type by its id."""
```

Reply:
May want to do a search and replace to fix all of them at once.

Reply:
Good catch. Done.